src/Core/src/Platform/iOS/NSAttributedStringExtensions.cs

```diff
@@ -37,8 +37,8 @@ public static class NSAttributedStringExtensions
 			if (lineHeight == -1 && attribute == null)
 				return null;
 
-			var mutableParagraphStyle = new NSMutableParagraphStyle(attribute);
-			mutableParagraphStyle.LineHeightMultiple = new nfloat(lineHeight >= 0 ? lineHeight : -1);
+			var mutableParagraphStyle = new NSMutableParagraphStyle();
```

Comment:
@hartez Sorry about the mistake here 😅. I was working blind in VS with no IntelliSense. You can add ```mutableParagraphStyle.SetParagraphStyle(attribute)``` right after this line so it copies whatever paragraph style was already there instead of overriding it with the default.